README.MD

```diff
@@ -458,7 +456,7 @@ import "./controllers/fooController.ts"
 @DefaultResponse<ErrorResponse>('Unexpected error')
```

Comment:
This should probably be updated

Reply:
What's the latest and greatest way to do this?

Reply:
I wish I could tell what the state of tsoa was when this was added.
When I got into it this was confusing though because this decorator [does not exist anymore](https://github.com/lukeautry/tsoa/blob/master/src/decorators/response.ts).

My best guess looking at the method is:

```suggestion
@SuccessResponse()
```

Reply:
With 
```ts
@Response('400', 'Bad request')
```

becoming:

```ts
@Response<ErrorResponse>('400', 'Bad request')
```